docs/tinlake/contracts/contracts.md

```diff
@@ -63,7 +63,7 @@ This method does a few calls:
 * Ceiling.repay(loan, wad) is called and needs to not revert to make the Ceiling aware of the decrease in debt
 
 ### Pile
-The default implementation of the Pile allows creating of different interest rate groups and assigning each loan a rate group. Each interest rate group has an interest rate that is calculated on a per second compounding basis.
+The default implementation of the `Pile` allows creating of different interest rate groups and assigning each loan a rate group. Each interest rate group has an interest rate that is calculated on a per second compounding basis.
```

Comment:
... ion of the `Pile` allows the creation of different interest rate groups and assigning each loan to a rate group...